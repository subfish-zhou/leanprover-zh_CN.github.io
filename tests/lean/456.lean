def A : Sort u := { s : Prop // _ }
