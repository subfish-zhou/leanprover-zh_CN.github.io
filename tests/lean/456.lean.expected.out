456.lean:1:18-1:35: error: failed to solve universe constraint
  u =?= max 1 ?u
while trying to unify
  Sort u : Type u
with
  Type : Type 1
